components/Body/Body.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "./Body.module.css";
+import SearchBar from "./SearchBar";
+import CardList from "../UI/Card/CardList";
+import FetchFolderData from "@/get-data/FetchFolderData";
+
+function Body() {
+  const folderdata = FetchFolderData();
+  return (
+    <div className={styles.body}>
+      <SearchBar />
+      {folderdata !== null ? (
+        <CardList />
+      ) : (
+        <div className={styles["no-links"]}>저장된 링크가 없습니다.</div>
+      )}
```

Comment:
```suggestion
      {folderdata ? (
        <CardList />
      ) : (
        <div className={styles["no-links"]}>저장된 링크가 없습니다.</div>
      )}
```
요렇게 코드를 작성하면 조금 더 직관적일 것 같아요 왜냐하면 folderData가 null이 아니다는 코드를 읽을때 deps가 한번 더 들어가서요!